setup.py

```diff
@@ -52,6 +52,7 @@
         "cloudpickle>=1.6.0,<=2.0.0",
         f"snowflake-connector-python{CONNECTOR_DEPENDENCY_VERSION}",
         "typing-extensions>=4.1.0, <5.0.0",
+        "pyyaml>=6.0",
```

Comment:
Required to parse Conda environment YAMLs, for adding requirements (as requested in the project JIRA).